qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -34,7 +34,7 @@
                                urlutils, message)
 
 
-UndoEntry = collections.namedtuple('UndoEntry', ['url', 'history', 'index'])
+UndoEntry = collections.namedtuple('UndoEntry', ['url', 'history', 'index', 'pinned'])
```

Comment:
Is there any way of getting the pinned status from a closed tab other than saving it to the undo stack? I'm not sure if any information is saved other than this. This seems a little to specific to be by itself on the stack in my opinion, but this is the cleanest way of adding it I found.